packages/xarc-app-dev/src/lib/utils.ts

```diff
@@ -156,3 +156,13 @@ export function detectCSSModule(xOptions) {
   }
```

Comment:
you need to update `loadXarcOptions` because it's using `process.cwd`, also do the same for its copy in `xarc-webpack`